src/olympia/shelves/migrations/0005_auto_20200807_1354.py

```diff
@@ -0,0 +1,27 @@
+# Generated by Django 2.2.15 on 2020-08-07 13:54
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('shelves', '0004_auto_20200805_1851'),
+    ]
+
+    operations = [
+        migrations.RemoveField(
+            model_name='shelf',
+            name='shelf_type',
+        ),
+        migrations.AddField(
+            model_name='shelf',
+            name='endpoint',
+            field=models.CharField(choices=[('collections', 'collections'), ('search', 'search')], max_length=200),
+        ),
+        migrations.AlterField(
+            model_name='shelf',
+            name='criteria',
+            field=models.CharField(help_text='e.g., ?sort=users&statictheme?recommended=true&sort=random&type=extensionpassword-managers', max_length=200),
+        ),
+    ]
```

Comment:
we don't really want to do this.  It'll delete the existing column for shelf_type and add a new one for endpoint - it's similar enough that we can just add `db_column='shelf_type'` in the `CharField` kwargs.  Then delete both these migrations and regenerate and there should be just a single migration with two AlterFields inside.